src/main/java/org/kohsuke/github/GHCommit.java

```diff
@@ -236,6 +170,7 @@ public URL getBlobUrl() {
         public String getSha() {
             return sha;
         }
+
     }
 
     /**
```

Comment:
Ugh, why is `Parent` (three lines below this one) public?  Not your fault.  Very odd.